src/components/ens-profile/RecordTags/RecordTags.tsx

```diff
@@ -132,6 +136,23 @@ function RecordTag({
   );
 }
 
+export function Hyperlink({ value }: { value: string }) {
```

Comment:
I'm not too sure if `Hyperlink` is in the same category as a "tag". In the future, we may want to add more styling to the tag which will also affect the hyperlink. Maybe we should make these components independant of eachother?

Reply:
Also, in the designs, the hyperlink is also the same font weight as the text